### PR TITLE
Make config defaults match description

### PR DESCRIPTION
**Pull request by utdrmac**
* **Analysis**: Reported on Slack by user AlexJ. configure.py does not set same defaults as the description for reactivate zeroed, and delegator pays.
* **Solution**: Correct logic
* **Performed tests**: See below

**Work effort**: 1.5hr

```
If a destination address has 0 balance, should burn fee be paid to reactivate? 1 for Yes, 0 for No. Type enter for Yes >

Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator >

Who is going to pay for transfer fees: 0 for delegator, 1 for delegate. Type enter for delegator >

Add any addresses with a special fee in form of 'PKH,fee'. Type enter to skip >

Add supporter address. Supporters do not pay bakery fee. Type enter to skip >

Configuration file is created at '/tmp/trd/pppp/tz1RMmSzPSWPSSaKU193Voh4PosWSZx1C7Hs.yaml'
utdrmac@blacklion:/tmp/trd$
utdrmac@blacklion:/tmp/trd$ cat pppp/tz1RMmSzPSWPSSaKU193Voh4PosWSZx1C7Hs.yaml
{baking_address: tz1RMmSzPSWPSSaKU193Voh4PosWSZx1C7Hs, delegator_pays_ra_fee: true,
    delegator_pays_xfer_fee: true, founders_map: {}, min_delegation_amt: 0.0, owners_map: {},
    payment_address: mybaker, reactivate_zeroed: true, rules_map: {mindelegation: TOB},
    service_fee: 10.0, specials_map: {}, supporters_set: !!set {}}
```

### CODE REVIEW

**Comment by utdrmac** [comment]
Another test:

```
Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator >
1
Who is going to pay for transfer fees: 0 for delegator, 1 for delegate. Type enter for delegator >
1
...
{baking_address: tz1RMmSzPSWPSSaKU193Voh4PosWSZx1C7Hs, delegator_pays_ra_fee: false,
    delegator_pays_xfer_fee: false, founders_map: {}, min_delegation_amt: 0.0, owners_map: {},

```

**Comment by utdrmac** [comment]
```
If a destination address has 0 balance, should burn fee be paid to reactivate? 1 for Yes, 0 for No. Type enter for Yes >
4
Invalid input: Please enter '0' or '1'
If a destination address has 0 balance, should burn fee be paid to reactivate? 1 for Yes, 0 for No. Type enter for Yes >
0
Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator >
66
Invalid input: Please enter '0' or '1'
Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator >
klasjdf
Invalid input: Please enter '0' or '1'
Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator >
1
```

File: src/configure.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
     'delegatorpaysrafee': "Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator",
     'supporters': "Add supporter address. Supporters do not pay bakery fee. Type enter to skip",
     'specials': "Add any addresses with a special fee in form of 'PKH,fee'. Type enter to skip",
-    'final': "Add excluded address in form of 'PKH,target'. Possbile targets are: TOB: leave at balance, TOF: to founders, TOE: to everybody. Type enter to skip"
```
Comment (jdsika) [inline src/configure.py:51]: why is this removed?

Comment (utdrmac) [reply src/configure.py:51]: Because it is not used. 'excluded addresses' is implemented 4 lines above.

**Review by jdsika — APPROVED** [review]
